### ISSUE TITLE
`tracing` feature has an implicit dependency on `futures` crate which should be declared

### ISSUE THREAD

**Issue by teohhanhui** [najamelan/async_nursery#4]
If the `tracing` feature of this crate is enabled without also enabling the `implementation` feature (e.g. `default-features = false`):

```
error[E0432]: unresolved import `futures`
 --> /home/teohhanhui/.cargo/registry/src/github.com-1ecc6299db9ec823/async_nursery-0.4.0/src/tracing.rs:5:2
  |
5 |     futures         :: { FutureExt                              } ,
  |     ^^^^^^^ use of undeclared crate or module `futures`
```

**Comment by najamelan** [comment]
Hi, thanks for reporting, I will look into this today and publish a new version if needed.

**Comment by najamelan** [comment]
I reviewed the dependencies and features and there where some issues. I released version 0.5 which also updates async_executors to the latest version and a 0.4.1 non breaking change for the 0.4 line.

Let me know if it works correctly for you now.
